### ISSUE TITLE
Native selection mode:NVDA+ double-clicking F10 does not copy format information

### ISSUE THREAD

**Issue by wmhn1872265132** [nvaccess/nvda#16073]
### Steps to reproduce:
1. Open any website in Firefox.
2. Enable Native selection mode.
3. Select a piece of content through NVDA+f9/F10.
4. Copy the content via NVDA + double-click F10.

### Actual behavior:

Format information is not copied

### Expected behavior:

Format information should be copied

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2024.1beta4
#### Windows version:
Windows 11 23H2 (AMD64) build 22631.3007
#### Name and version of other software in use when reproducing the issue:
Mozilla Firefox 121.0.1 (64-bit)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by wmhn1872265132** [comment]
It should be pointed out that you can copy the format information by first selecting any content through NVDA+F9/F10, and then pressing Ctrl+C to copy.

**Comment by cary-rowen** [comment]
This is a bit counter-intuitive, often I would think that having performed the selection in native selection mode, the copy should also include the formatting.

**Comment by Adriani90** [comment]
cc: @jcsteh is this at all possible? I guess the browsers intercept only the ctrl+c command right?
The NVDA+f10 twice command isn NVDA specific command and I am not sure this is being sent to the application like ctrl+c.

**Comment by jcsteh** [comment]
The only way this could be done is to have NVDA+f10 twice send control+c to the application in this case. There is an IAccessibleEditableText::copyText API, but that function takes offsets within a single object which is not sufficient to represent a selection across objects.

**Comment by seanbudd** [comment]
>The only way this could be done is to have NVDA+f10 twice send control+c to the application in this case. There is an IAccessibleEditableText::copyText API, but that function takes offsets within a single object which is not sufficient to represent a selection across objects.

The issue with that is the keyboard shortcut may be different particularly depending on the locale

**Comment by jcsteh** [comment]
Fair, but we already have that problem with CursorManager.script_copyToClipboard. It assumes that NVDA's key binding matches the app's key binding for copy (which is not guaranteed) and just [sends the gesture through](https://github.com/nvaccess/nvda/blob/4bd5a3dd4dcf705e244efe842e8209fe64757443/source/cursorManager.py#LL536).
